x-pack/plugins/observability_solution/observability_onboarding/public/application/experimental_onboarding_flow.tsx

```diff
@@ -13,32 +20,72 @@ import { Footer } from './footer/footer';
 import { OnboardingFlowForm } from './onboarding_flow_form/onboarding_flow_form';
 import { Header } from './header/header';
 
+const queryClient = new QueryClient();
+
+interface Props {
+  selectedCategory?: string;
+}
+
+function OnboardingFlowPackageList({
+  selectedCategory = 'observability',
+}: Props) {
+  const availablePackages = useAvailablePackages({
+    prereleaseIntegrationsEnabled: false,
+  });
+  return (
+    <EuiPanel>
+      <PackageListGrid
+        list={availablePackages.filteredCards.filter((card) =>
+          card.categories.includes(selectedCategory)
+        )}
+        searchTerm={''}
+        setSearchTerm={() => {}}
+        selectedCategory={selectedCategory}
+        setCategory={() => {}}
+        categories={[]}
+        setUrlandReplaceHistory={() => {}}
+        setUrlandPushHistory={() => {}}
+        showSearchTools={false}
+      />
+    </EuiPanel>
+  );
+}
+
 export function ExperimentalOnboardingFlow() {
   return (
-    <>
-      <EuiPageTemplate.Section
-        paddingSize="xl"
-        css={css`
-          & > div {
-            background-image: url(${backgroundImageUrl});
-            background-position: right center;
-            background-repeat: no-repeat;
-          }
-        `}
-        grow={false}
-        restrictWidth
-      >
-        <EuiSpacer size="xl" />
-        <Header />
-      </EuiPageTemplate.Section>
-      <EuiPageTemplate.Section paddingSize="xl" color="subdued" restrictWidth>
-        <OnboardingFlowForm />
-        <EuiSpacer size="xl" />
-      </EuiPageTemplate.Section>
-      <EuiPageTemplate.Section paddingSize="xl" grow={false} restrictWidth>
-        <Footer />
-        <EuiSpacer size="xl" />
-      </EuiPageTemplate.Section>
-    </>
+    <QueryClientProvider client={queryClient}>
+      <EuiThemeProvider darkMode={false}>
+        <>
+          <EuiPageTemplate.Section
+            paddingSize="xl"
+            css={css`
+              & > div {
+                background-image: url(${backgroundImageUrl});
+                background-position: right center;
+                background-repeat: no-repeat;
+              }
+            `}
+            grow={false}
+            restrictWidth
+          >
+            <EuiSpacer size="xl" />
+            <Header />
+          </EuiPageTemplate.Section>
+          <EuiPageTemplate.Section
+            paddingSize="xl"
+            color="subdued"
+            restrictWidth
+          >
+            <OnboardingFlowForm />
+            <EuiSpacer size="xl" />
+            <OnboardingFlowPackageList />
```

Comment:
Is there a reason to render the package list outside of the form? In the design it's in the same panel and we need to change the filter based on the form selection so it might be easier to move it into the form. 

The spacing below the form was intentional to separate the content from the footer correctly so I would keep that in.